openwisp_radius/api/serializers.py

```diff
@@ -78,6 +86,12 @@ def _disable_token_auth(self, user):
             radius_token.can_auth = False
             radius_token.save()
 
+    def run_validation(self, data):
+        for field in ['session_time', 'input_octets', 'output_octets']:
+            if data.get('status_type', None) == 'Start' and data[field] == '':
+                data[field] = 0
```

Comment:
This will get the job done. `validate()` would have not worked because `run_validate` is done before `validate()`.

Reply:
Great!